### PR TITLE
GitHub webhook

### PR DESCRIPTION
**Pull request by bkuhl**
* Back to nginx/fpm in the same container
 * Updated webhook to be compatible with GitHub
 * Incorporated @jlaswell's feedback from the other PR in regards to container structure

### CODE REVIEW

File: infrastructure/start.sh [hunk 1]
```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+echo "Setting environment variables for php-fpm..."
```
Comment (jlaswell) [inline infrastructure/start.sh]: Do we need this `start.sh` file too?

**Comment by jlaswell** [comment]
Nice work!! LGTM after `start.sh` is deleted.

**Comment by bkuhl** [comment]
Unless you are seeing something I'm not, it's not possible to delete that script because it handles environment variables and the SSH key. So it has to happen at runtime.

**Comment by bkuhl** [comment]
I was looking at this on my phone.  Looking at this on a Desktop I see the duplicate code now.  `env_init.sh` has been removed.

**Pull request merged**